packages/esm-patient-common-lib/src/print/print.component.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react';
+import { Printer } from '@carbon/react/icons';
+import { useReactToPrint } from 'react-to-print';
+import { useTranslation } from 'react-i18next';
+import styles from './print.scss';
+import { Header, Button } from '@carbon/react';
+import log from './log.png';
+
+export function PrintComponent({ subheader, body, patientDetails, requestedBy }) {
```

Comment:
Do we have an example usage of how these props should be populated?  For example, is "patientDetails" a standard data structure we define or us elsewhere?